Code/Mantid/Framework/Algorithms/test/ConvertToMatrixWorkspaceTest.h

```diff
@@ -59,22 +63,24 @@ class ConvertToMatrixWorkspaceTest : public CxxTest::TestSuite
     checker.setPropertyValue("Workspace2","out");
```

Comment:
Use setProperty where possible rather than setProperyValue. You only use setPropertyValue because you've put the workspace in the AnalysisDataService.